app/models/media_object.rb

```diff
@@ -747,6 +759,13 @@ def remove_bookmarks
     self._remove_bookmarks
   end
 
+  # Setters that should go through update_datastream
+  [ :genre ].each do |setter|
```

Comment:
I like your instinct here, but would it additionally be worth extracting [:genre] into a class-level constant so people there's a clear point for modification in the future?